lib/matplotlib/axes/_base.py

```diff
@@ -4477,6 +4477,8 @@ def get_tightbbox(self, renderer, call_axes_locator=True,
         """
 
         bb = []
+        if renderer is None:
+            renderer = self.figure._get_cached_renderer()
```

Comment:
Is there a reason to put this inside `get_tightbbox()` versus `get_window_extent()`? It looks like further down you put it inside the `get_window_extent()` call, so I wasn't following where this is needed versus where `None` could be passed through...

Reply:
The locator needs the renderer, however, its debatable whether we should remove that as well.